.jvmopts

```diff
@@ -1,4 +1,5 @@
 -Dfile.encoding=UTF8
+-Djava.security.egd=file:/dev/./urandom
```

Comment:
Good find, too bad this isn't the default as AFAIK this is not any less secure on modern systems (https://www.2uo.de/myths-about-urandom/)